frontend/src/components/SimpleRange/SimpleRange.js

```diff
@@ -1,18 +1,18 @@
-import { FormattedNumber } from 'react-intl'
 import { styled } from '@mui/material/styles'
 import LinearProgress, { linearProgressClasses } from '@mui/material/LinearProgress'
-import Tooltip from '@mui/material/Tooltip'
 import PropTypes from 'prop-types'
 
 const GradientProgress = styled(LinearProgress)(({ theme, value }) => ({
   [`& .${linearProgressClasses.bar}`]: {
     overflow: 'hidden',
     backgroundColor: 'transparent',
+    borderRadius: 1,
     '&:before': {
       content: '""',
       display: 'block',
       width: '100%',
       height: '100%',
+      borderRadius: 1,
```

Comment:
The `borderRadius: 1` on .bar is rendered as `border-radius: 4px`, but the `borderRadius: 1` on ::before is rendered as `border-radius: 1px` instead of the intended 4px. That seems like a bug in MUI to me, but I don't have much experience with it. I would suggest hardcoding the ::before borderRadius as 4px.